m2m-transfers/V1/m2m-transfers-json-contract.md

```diff
@@ -332,19 +332,53 @@ EDITTRAN - Edit existing transaction (expire existing transfer & create a new tr
 }
 ```
 ### Error Codes
-501 - Config file read / validation error
-502 - Invalid source account
-503 - Invalid recipient account
-504 - Insufficient information
-505 - Invalid input
-506 - Error creating/deleting recipient
-507 - Error creating/updating/deleting transfer
-508 - Undefined error
-509 - Member unverified
-510 - AccountId incorrect
+
+*See Logging Error Message section for additional details
+
+| Error Code | Logging Error Message                                                       | Notes |
+| -----------|------------------------------------------------------------------------------------- |-----------------------------------------------|
+| 500        | Program running in memo mode ||
+| 501        | Config file error: [configuration file name] open error - [system generated letter file read error message] | This error represents issues with the configuration letter file setup |
+|            | Config file error: [configuration file name] read error - [system generated letter file read error message] ||
+|            | Config file error: 3-Error Reading Letterfile [configuration file name]: [system generated letter file read error message] ||
+|            | Config file error: Invalid Parameter in CFG file ||
+| 502        | Invalid Source Account: Pref Access type 3 not found ||
+|            | Invalid Source Account: Acct Warning 1234 ||
+|            | Invalid Source Account: No eligible transfer from shares/loans ||
+|            | Invalid Source Account: Acct Type 1234 ||
+| 503        | Invalid Recipient Account ||
+| 504        | Invalid Source AccountCannot calc member limits. ||
+| 505        | Invalid Input ||
+| 506        | Error Processing Recipient Record | Error creating/deleting recipient |
+| 507        | Error Processing Transfer Record | Error creating/updating/deleting transfer |
+|            | Error Processing Transfer RecordTarget s/l xfer Loc 1234567 not found ||
```

Comment:
Missing Colon

Reply:
Same reason as my comment above:
"
Thanks Justin. The missing colon is because the code for V1 M2M is that way, and Emily and Tiffany told Tom not to update V1. However; I've moved that part of the comment into a new column called modifier.

I've been updating the code on my laptop. In a little bit I will send a second commit PR with all the recommended changes.

"